styles/layout.scss

```diff
@@ -0,0 +1,196 @@
+.shop-item {
+  border: $c-primary solid 2px;
+
+}
+
+.shop-item__details {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  color: $c-primary;
+  text-align: center;
+
+}
+
+.shop-item__actions {
+  display: flex;
+  flex-direction: row;
+  // position: absolute;
+}
+
+.shop-item__frame {
+  margin: $c-gutter;
+}
+
+.shop-item__actions-btn {
+  display: flex;
+  justify-content: center;
+  flex-grow: 1;
+
+  border: $c-primary 1px solid;
+  border-bottom: none;
+  border-left: none;
+
+  &:last-of-type {
+    border-right: none;
+  }
+
+  padding: 1rem;
+  background-color: $c-white;
+  color: $c-primary;
+
+  &:hover {
+
+    background-color: $c-primary;
+    color: $c-white;
+
+  }
+
+}
+
+.shop-item__actions-btn {
+  position: relative;
+}
+
+.shop-menu {
+  &__tooltip {
+    display: none;
+    position: absolute;
+
+    flex-direction: column;
+    bottom: 2rem;
+
+    &:hover {
+
+      background: $c-white;
+      color: $c-primary;
+    }
+  }
+
+  &:hover {
+    color: $c-white;
+
+    .shop-menu__tooltip {
+      display: flex;
+      background: $c-white;
+      color: $c-primary;
+
+    }
+  }
+}
+
+.shop-menu__tooltip-item {
+
+  padding: 1rem;
+  border: $c-primary 2px solid;
+
+  &:hover {
+
+    background-color: $c-white;
+    height: 3.1rem;
+
+  }
+
+  &--color-red {
+    color: $c-pink;
+
+    &:hover {
+      background-color: $c-white;
+      height: 3.1rem;
+
+    }
+  }
+
+  &--color-blue {
+    color: $c-blue;
+
+    &:hover {
+      background-color: $c-white;
+      height: 3.1rem;
+
+    }
+  }
+
+  &--color-yellow {
+    color: $c-yellow;
+
+    &:hover {
+      background-color: $c-white;
+      height: 3.1rem;
+
+    }
+  }
+
+  &--color-green {
+    color: $c-green;
+
+    &:hover {
+      background-color: $c-white;
+      height: 3.1rem;
+
+    }
+  }
+
+}
+
+//grid-flex starts @ line 13, ends @ 244
+@media (max-width: 768px) and (min-width: 480px) {
+  .grid {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+
+    flex-basis: calc(100% / 3 - 1rem);
+
+  }
+
+  &__item {
+    display: flex;
```

Comment:
Look at moving display flex to the regular mode. This will help since you'll be able to use `align-items` to center content.

That makes it so that the only thing that changes in the media query is the direction 
<img width="1680" alt="screen shot 2016-10-12 at 2 36 58 pm" src="https://cloud.githubusercontent.com/assets/2532004/19325217/b44c3b2a-908a-11e6-931f-703989c3d4ff.png">

Notice in that screenshot how the tshirts aren't centered.